manim/mobject/geometry.py

```diff
@@ -757,10 +757,34 @@ def __init__(self, outer_radius=1, inner_radius=0, **kwargs):
 
 
 class Annulus(Circle):
+    """Region between two concentric :class:`Circle`s.
```

Comment:
This unfortunately doesn't render correctly, ReST doesn't really allow letters directly following links:

![image](https://user-images.githubusercontent.com/11851593/120232710-63a91980-c254-11eb-8aed-5a34f4f34315.png)

Here is a potential workaround:

```suggestion
    """Region between two concentric :class:`Circles <.Circle>`.
```

Reply:
Oh, sorry